service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/OrdersNeededController.java

```diff
@@ -152,4 +162,21 @@ private void removeSecureAccommodationOrderFields(Map<String, Object> data, Stri
     private Court getCourtSelection(String courtID) {
         return courtLookup.getCourtByCode(courtID).orElse(null);
     }
+
+    private boolean isContactWithChildInCareOrder(Optional<List<String>> orderType) {
+        return orderType.isPresent()
```

Comment:
there is a new method caseData.isContactWithChildInCareApplication. Can we use it instead of duplicating similar logic?